src/accountsdb/fuzz.zig

```diff
@@ -145,6 +155,17 @@ pub fn run(seed: u64, args: *std.process.ArgIterator) !void {
     var pubkeys_this_slot = std.AutoHashMap(Pubkey, void).init(allocator);
     defer pubkeys_this_slot.deinit();
 
+    for (0..8) |i| {
+        _ = try std.Thread.spawn(.{}, readRandomAccountThread, .{
```

Comment:
should `.detach()` the thread handles to avoid leaking their resources.

Reply:
addressed in https://github.com/Syndica/sig/commit/972fb916f818893e5c4f7f64db5ee9dbe660a322